MapboxNavigation-Documentation.podspec

```diff
@@ -47,7 +47,7 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxDirections.swift", "~> 0.30.0"
   s.dependency "MapboxGeocoder.swift", "~> 0.10.0"
-  s.dependency "Mapbox-iOS-SDK", "~> 5.5.0"
+  s.dependency "Mapbox-iOS-SDK", "= 5.5.0"
   s.dependency "MapboxMobileEvents", "~> 0.9.5"
```

Comment:
Should we also pin MapboxMobileEvents to v0.9.5 specifically and avoid patch releases? Unlike with the map SDK, this branch is still on a pre-v1.0 release, so it wouldn’t float to v0.10._x_.

/cc @rclee @alfwatt